libdevcore/CommonData.h

```diff
@@ -50,14 +50,27 @@ template <class T, class U> std::vector<T>& operator+=(std::vector<T>& _a, U&& _
 	std::move(_b.begin(), _b.end(), std::back_inserter(_a));
 	return _a;
 }
+/// Concatenate the contents of a container onto a multiset
+template <class U, class... T> std::multiset<T...>& operator+=(std::multiset<T...>& _a, U const& _b)
+{
+	_a.insert(_b.begin(), _b.end());
+	return _a;
+}
+/// Concatenate the contents of a container onto a multiset, move variant.
+template <class U, class... T> std::multiset<T...>& operator+=(std::multiset<T...>& _a, U&& _b)
+{
+	for (auto&& x: _b)
+		_a.insert(std::move(x));
```

Comment:
Why not use ``std::forward<U>(x)` here? In that case, I think we would only need one function.

Reply:
Ah sorry, my bad, this is not `_b` and we even discussed this at length...